examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -1,5 +1,21 @@
 # Matter nRF Connect All Clusters Example Application
 
+> **Note:** This example is intended only to perform smoke tests of a Matter
```

Comment:
As this Note is a new thing, is it possible to add that this is introduced from Matter x.x or NCS x.x.x version?

Reply:
I would not like to do that. It would suggest that before the Matter x.x or NCS x.x.x it was recommended to use this examples instead of the ones located in NCS. We have always recommended to use samples from NCS as a reference, while this examples were only to perform the newest features integration as soon as possible and we couldn't promise they are production ready.

Unfortunately this was a mistake from our side, that we haven't communicated it in a more clear way from the beginning. But we do see customers that struggle with multiple problems using this examples, that are not present in samples from NCS, so we would like to try to minimize the risk and improve the user experience for our platform.